README.md

```diff
@@ -31,12 +31,21 @@ If upgrading to version v1.10.0, you may end up with duplicate installations of
 
 ## Compatibility
 
-The current package is confirmed to be working in different Synology models and architectures.
+The current package is confirmed to be working in different Synology models and architectures. [Some features](#synology-limitations) are currently not available.
 
 The package is created based on Tailscale [static binaries](https://pkgs.tailscale.com/stable/#static), and if your NAS has any of the supported architectures (x86, x86_64, arm, arm64) it should _just_ work.
 
 If in doubt, check the [synology model list](docs/platforms.md) for the matching architecture.
 
+### Synology Limitations
+
+Ref: [Issue #1995](https://github.com/tailscale/tailscale/issues/1995)
+
+Things currently not supported on Synology are:
+
+- any tailscale up `--netfilter-mode=XXX` value other than `off`.
+- To enable exit node/subnet router, consider using the `--reset` command, e.g. `sudo tailscale up --advertise-routes=192.168.1.0/24 --advertise-exit-node --reset`. The non-default setting `--accept-routes` is not supported on synology.
```

Comment:
I suppose this might be a bug with synology package: If I understood the error message it is saying the newly deploy tailscale instance on synology was running with `--accept-routes` - dropping that flag and adding `--reset` worked so i thinking maybe default launch setting for synology needs that `--accept-routes` removed.